experiment/autobumper/bumper/bumper.go

```diff
@@ -286,6 +286,17 @@ func Run(o *Options) error {
 			}
 		}
 
+		// Check to see if the proper fork exists and if it does not, create one.
+		// TODO(mpherman): Handle case where account has repo of same name as one we are trying to autobump but is not a fork of it (e.g. test-infra)
+		_, err := gc.GetRepo(o.GitHubLogin, o.GitHubRepo)
```

Comment:
Can we do better error checking to differentiate from repo not exist vs. random errors? If the error type is not easy to get, maybe try use ListRepo then filter out the names(need to be careful of pagination)?

Reply:
Done. I originally tried to get just the error code (404) but that isn't piped to the response as far as I can tell, but I got it working with the error string instead.